src/backend/api/Fusion.Resources.Database/Migrations/20210317154326_Added subtype to request.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Fusion.Resources.Database.Migrations
+{
+    public partial class Addedsubtypetorequest : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "SubType",
+                table: "ResourceAllocationRequests",
+                type: "nvarchar(max)",
```

Comment:
I guess this is nitpicking, and a problem many other places, but we should probably avoid using nvarchar(max) for short, known(ish) length strings as nvarchar(max) values are stored outside the row in the database and requires more reads for each row. This is especially the case if we want to index the field.

Reply:
Yeh, it's not even nitpicking.. forgot about this.

Reply:
I think we could probably do that in a separate pull request, for many different entities